ui/app/components/app-form/project-settings.hbs

```diff
@@ -0,0 +1,163 @@
+<form class="pds-form">
+  <div part="fields">
+    <fieldset class="pds-formFieldSet">
+      <div class="pds-formField">
+        <label for="git-source-url" class="pds-fieldName">
+          Git Source URL
```

Comment:
Are y'all using `ember-intl` ? I saw there was a `translations/` folder but didn't dig too much further than that. If you want to linting to be loud about it you can turn on `no-bare-strings` rule in `ember-cli-template-lint`.

Reply:
done!